manifests/pipecd/values.yaml

```diff
@@ -29,7 +29,7 @@ gateway:
 
 server:
   image:
-    repository: ghcr.io/pipe-cd/pipecd
+    repository: ghcr.io/pipe-cd/control-plane
```

Comment:
This should be change later, by separated PR when we introduce control-plane helm chart, since it's the manifest for installing pipecd control-plane via helm